app/src/main/java/cloudcity/CloudCityParamsRepository.java

```diff
@@ -8,9 +8,16 @@
 
 import androidx.annotation.Nullable;
 
+import de.fraunhofer.fokus.OpenMobileNetworkToolkit.BuildConfig;
+
+
 public class CloudCityParamsRepository {
     private static final String SHARED_PREFS_NAME = "cloud_city_prefs";
 
+    private static final String STAGING_SERVER_URL = "staging.app.cloudcities.co";
+    private static final String PRODUCTION_SERVER_URL = "app.cloudcities.co";
+    private static final String DEMO_SERVER_URL = "demo.app.cloudcities.co";
```

Comment:
no, the Retrofit will format the base URL to look like this:
```
"https://%s/"
```
so it will end up being what you suggested - `https://<flavor-dependant server url>/`

Reply:
As mentioned before (what I also told you to learn) - try to grasp the bigger context of the change. resolving.